azure-pipelines-docs-integration.yml

```diff
@@ -109,10 +109,10 @@ stages:
               retryCount: '2'
 
         - script: |
-            curl -sS https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-367.0.0-linux-x86_64.tar.gz > ./google-cloud-sdk-367.0.0-linux-x86_64.tar.gz && tar -xf ./google-cloud-sdk-367.0.0-linux-x86_64.tar.gz
+            curl -sS https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-370.0.0-linux-x86_64.tar.gz > ./google-cloud-sdk-370.0.0-linux-x86_64.tar.gz && tar -xf ./google-cloud-sdk-370.0.0-linux-x86_64.tar.gz
             ./google-cloud-sdk/install.sh --usage-reporting=False --path-update=True --quiet --install-python=False
-            ./google-cloud-sdk/bin/gcloud config configurations create superconductive-internal-configurations
-            ./google-cloud-sdk/bin/gcloud config set account sandbox-account-for-azure-tests --project=superconductive-internal --access-token-file=$(gcp_authkey.secureFilePath)
+            ./google-cloud-sdk/bin/gcloud config configurations create ge-oss-ci-cd-configurations
+            ./google-cloud-sdk/bin/gcloud config set account -account-for-azure-tests --project=$(GE_TEST_GCP_PROJECT) --access-token-file=$(gcp_authkey.secureFilePath)
             ./google-cloud-sdk/bin/gcloud config set pass_credentials_to_gsutil True
             ./google-cloud-sdk/bin/gcloud auth activate-service-account --key-file=$(gcp_authkey.secureFilePath)
```

Comment:
Is this the recommended script from the official docs? If so, could we please add a comment at the top of this section that points to that ref?

If this is a custom script, could we annotate each part or major section with comments? This seems quite complicated so I'm thinking of ways to make it easy to debug.

Something like this could work: https://github.com/great-expectations/great_expectations/blob/78dc797ba5036c7f02e1bf32ba02da1945107170/azure-pipelines.yml#L83

Alternatively, I think wrapping all this logic in a Python script so we can add relevant printing/logging is my preferred approach. Using `subprocess`, we can run a list of commands and check status codes/print to stdout at each step.

Let me know your thoughts and if you want to discuss/pair.

Reply:
I like the python-script approach better too :) Let me put something together